shared/js/ui/models/mixins/parse-user-agent.es6.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
   parseUserAgentString (uaString) {
     if (!uaString) uaString = window.navigator.userAgent
-    const rgx = uaString.match(/(Firefox|Chrome)\/([0-9]+)/)
+    const rgx = uaString.match(/(Firefox|Chrome)\/([0-9.]+)/)
+
     return {
       browser: rgx[1],
-      majorVersion: rgx[2]
+      version: rgx[2]
```

Comment:
`majorVersion` wasn't really used anywhere.

Figured we might as well grab the whole thing.